### DISCUSSION TITLE
Feature Request: Show process launch information

### DISCUSSION THREAD

**Discussion by BlakeTheAwesome** [microsoft/terminal#17153] [Ideas]
I have the terminal set to not auto-close once a process has finished, but at startup I end up with a few empty tabs that were launched by something, but I can't easily track down what started them.
It would be nice if I could right-click on a tab or something, and find out the information that the console was launched with (command line args, working dir, etc.)

**Comment by zadjii-msft** [comment]
FWIW You can enable logging for process creation within Event Viewer itself. Here's a guide: https://logrhythm.com/blog/how-to-enable-process-creation-events-to-track-malware-and-threat-actor-activity/

That's not as straightforward as doing it directly in the Terminal, no, but it should probably work for your use case
